packages/prisma/schema.prisma

```diff
@@ -598,8 +600,10 @@ model WorkflowStep {
 model Workflow {
   id       Int                     @id @default(autoincrement())
   name     String
```

Comment:
Should store `owner` here, otherwise we would never know who creates this workflow

Reply:
I am now always saving the user who created the workflow even when it is a team workflow (https://github.com/calcom/cal.com/pull/7038/commits/b22d0ba0347d51223e188a20e4657fb9034702e4)